src/deprecates.jl

```diff
@@ -10,3 +10,7 @@ import Base.depwarn
 @deprecate se(x::AbstractVector) stderror(x)
 @deprecate se(bs::BootstrapSample) stderror(bs)
 @deprecate se(bs::BootstrapSample, idx::Int) stderror(bs, idx)
+
+# functions before data
+@deprecate bootstrap(data, statistic::Function, sampling::Sampling) where Sampling<:BootstrapSampling bootstrap(statistic, data, sampling)
```

Comment:
Just FYI, the `where` clause isn't necessary here; you can simply write this as `sampling::BootstrapSampling`.

Reply:
Ah, thanks! This looks much nicer [now](1daaf12a1f908c42a6032e672aa2be22c83f3e41).